integration/test-integration/src/test/resources/dockerCompose/apim-in-common-network-docker-compose.yaml

```diff
@@ -4,16 +4,16 @@ networks:
     name: apim_and_cc
 services:
   apim:
-    image: wso2/wso2am:4.0.0-alpine
+    image: wso2am/wso2am:4.1.0-alpha-alpine
     user: root
     ports:
       - "9763:9763"
       - "9443:9443"
       - "5672:5672"
       - "8243:8243"
     volumes:
-      - ./conf/deployment.toml:/home/wso2carbon/wso2am-4.0.0/repository/conf/deployment.toml
-      - ./database:/home/wso2carbon/wso2am-4.0.0/repository/database:rw
+      - ./conf/deployment.toml:/home/wso2carbon/wso2am-4.1.0-alpha/repository/conf/deployment.toml
+      - ./database:/home/wso2carbon/wso2am-4.1.0-alpha/repository/database:rw
```

Comment:
is this change required in the file [resources/docker-compose/apim/docker-compose.yaml](https://github.com/wso2/product-microgateway/pull/2714/files#diff-aa496dca1149563958a1b496c03c6258cda6a3536eb9e83e41b5880061e749a2) too, i think i forgot